src/main/java/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -638,13 +638,8 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
   }
 
   static Match<Unit> attackingGroundSeaBattleEscorts(final PlayerID attacker, final GameData data) {
-    return new Match<Unit>() {
-      @Override
-      public boolean match(final Unit u) {
-        final Match<Unit> canIntercept = Match.all(Matches.unitCanAirBattle);
-        return canIntercept.match(u);
-      }
-    };
+    final Match<Unit> canIntercept = Matches.unitCanAirBattle;
+    return canIntercept;
```

Comment:
Understood.  I just couldn't reconcile the name of this method with what it was doing, so I didn't want to remove any additional context in the event someone else was just as confused as me. :smile: